src/js/_enqueues/wp/theme.js

```diff
@@ -1082,10 +1082,10 @@ themes.view.Themes = wp.Backbone.View.extend({
 		// Update theme count to full result set when available.
 		this.listenTo( self.collection, 'query:success', function( count ) {
 			if ( _.isNumber( count ) ) {
-				self.count.text( count );
+				this.renderThemeCount( count );
```

Comment:
`this.renderThemeCount( count )` can be moved outside of the if-else statement since both calls have the same arguments.

Reply:
In fact, it can be simplified to:
```
this.renderThemeCount( count );

self.announceSearchResults( _.isNumber( count ) ? count : self.collection.length );
```